### DISCUSSION TITLE
能否在评论区把被B站刻意沉底的热评捞上来，或强制按赞数重新排列？谢谢！

### DISCUSSION THREAD

**Discussion by 4AMNyaruko** [the1812/Bilibili-Evolved#3446] [功能建议 / Ideas]
例：https://t.bilibili.com/674080484363337784
捷豹收到针对（当然还有一些别的up主）！捷豹几乎的所有动态评论区超过一定赞数的热评都会被B站强行“沉底”（按热度排序，拉到底可见热评没有被删而是沉底），能否有脚本能把这些热评捞上去，强制按点赞数重新排列？谢谢大佬们！
PS：热评沉底现象多发生于一些特定的up主，他们的动态评论会发生这样的现象，他们在其他视频所发的评论也会被沉底（或被该视频up主主动沉底），不知道能否对评论区楼层进行重新排序？谢谢！

**Comment by 4AMNyaruko** [comment]
![截屏2022-06-21 16 28 56](https://user-images.githubusercontent.com/107914878/174754088-84a4ca7c-76ed-4b58-9b14-7e3b283d50b9.png)
![截屏2022-06-21 16 28 22](https://user-images.githubusercontent.com/107914878/174754114-35fe5869-3fad-4a00-bc19-f667d9409f5d.png)
可见高赞评论和有多条回复的评论全部被沉底了。

**Comment by the1812** [comment]
捞不了

**Comment by FCSDFSDVXDFG** [comment]
这应该是B站改了热评机制而不是刻意沉底，很早就知道了，暂不清楚是什么算法，反正高赞和无赞评论都看得到
之前热评被改成靠点赞速度而不是数量来排序，就一堆人问“你赞这么少怎么上来的”，点赞速度来排序本来也能接受，没想到现在B站改成这么抽象的排序机制了
我之前也想过提建议让脚本作者把评论排序改一下，但仔细一想很难做到
